network-conversion-server/src/main/resources/application.yml

```diff
@@ -0,0 +1,21 @@
+server:
+  port: 5003
+
+spring:
+  servlet:
+    multipart:
+      max-file-size: 100MB
+      max-request-size: 103MB
+
+network-store-server:
+  preloading-strategy: COLLECTION
+  base-uri: http://localhost:8080/
```

Comment:
We should remove this from the application.yml shipped in the jar, so that the default host values are used